common/flatpak-dir-private.h

```diff
@@ -92,6 +92,12 @@ GType flatpak_deploy_get_type (void);
 #define FLATPAK_REPO_DEPLOY_COLLECTION_ID_KEY "DeployCollectionID"
 #define FLATPAK_REPO_DEPLOY_SIDELOAD_COLLECTION_ID_KEY "DeploySideloadCollectionID"
 
+#define FLATPAK_PREINSTALL_GROUP "Flatpak Preinstall"
```

Comment:
Why is this in the header and not in the c file?

Reply:
I just put it in the same file that defined FLATPAK_REF_GROUP to keep things organized similarly, but beyond that I don't see any reason why it couldn't be in the .c file instead, indeed.